packages/renderers/lib/src/request/request_item_renderer/consent_request_item_renderer.dart

```diff
@@ -1,34 +1,210 @@
 import 'package:enmeshed_types/enmeshed_types.dart';
+import 'package:enmeshed_ui_kit/enmeshed_ui_kit.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_i18n/flutter_i18n.dart';
 import 'package:get_it/get_it.dart';
+import 'package:go_router/go_router.dart';
+import 'package:i18n_translated_text/i18n_translated_text.dart';
 
 import '../../abstract_url_launcher.dart';
-import '../../custom_list_tile.dart';
+import '../request_item_index.dart';
+import '../request_renderer_controller.dart';
+import 'decidable/checkbox_enabled_extension.dart';
+import 'decidable/widgets/handle_checkbox_change.dart';
+import 'decidable/widgets/validation_error_box.dart';
 
-class ConsentRequestItemRenderer extends StatelessWidget {
+class ConsentRequestItemRenderer extends StatefulWidget {
   final ConsentRequestItemDVO item;
+  final RequestRendererController? controller;
+  final RequestItemIndex itemIndex;
 
-  const ConsentRequestItemRenderer({super.key, required this.item});
+  final RequestValidationResultDTO? validationResult;
+
+  const ConsentRequestItemRenderer({super.key, required this.item, this.controller, required this.itemIndex, this.validationResult});
+
+  @override
+  State<ConsentRequestItemRenderer> createState() => _ConsentRequestItemRendererState();
+}
+
+class _ConsentRequestItemRendererState extends State<ConsentRequestItemRenderer> {
+  late bool _isChecked;
+
+  @override
+  void initState() {
+    super.initState();
+
+    _isChecked = widget.item.initiallyChecked || widget.item.response is AcceptResponseItemDVO;
```

Comment:
So let's say we get this request as a message and `widget.item.initiallyChecked = true`, then the person rejects this.. It would show up as a response with it still being checked. Isn't that confusing ? Or do we just miss a text like "you rejected this request" in the `message_detail_screen.dart` ? 

Otherwise, I might suggest something like this:
```suggestion
    if (widget.item.response != null) {
      _isChecked = widget.item.response is AcceptResponseItemDVO;
    } else {
      _isChecked = widget.item.initiallyChecked;
    }
```

Reply:
ehrm, this is a very good question...